solr/core/src/test/org/apache/solr/handler/admin/AdminHandlersProxyTest.java

```diff
@@ -91,6 +98,32 @@ public void proxyMetricsHandlerAllNodes() throws IOException, SolrServerExceptio
     assertNotNull(((NamedList)nl.get(nl.getName(1))).get("metrics"));
   }
 
+  @Test
+  public void proxyLoggingHandlerAllNodes() throws IOException, SolrServerException {
+    CollectionAdminRequest.createCollection("collection", "conf", 2, 2).process(solrClient);
+
+    mparams = new ModifiableSolrParams();
+    mparams.set(CommonParams.QT, "/admin/logging");
+    mparams.set("nodes", "all");
+    mparams.set("set", "com.codahale.metrics.jmx.JmxReporter:WARN");
+    req = new GenericSolrRequest(SolrRequest.METHOD.GET, "/admin/logging", mparams);
+    req.process(solrClient, "collection");
+
+    Set<String> nodes = solrClient.getClusterStateProvider().getLiveNodes();
+    nodes.forEach(node -> {
+      mparams.clear();
+      mparams.set("nodes", node);
+      req = new GenericSolrRequest(SolrRequest.METHOD.GET, "/admin/logging", mparams);
+      try {
+        rsp = req.process(solrClient, "collection");
+      } catch (Exception e) {
+        fail("Exception while proxying request to node " + node);
```

Comment:
If above you used a standard for loop, you wouldn't be forced to catch this exception, and it'd be more succinct.  Personally, I only use Collection.forEach lambda for a one-liner or something short and no exception propagation concern.